### ISSUE TITLE
Feature Request: Add `access` (, `motor_vehicle`) property on `streets`

### ISSUE THREAD

**Issue by westnordost** [shortbread-tiles/shortbread-docs#38]
The information whether a road is accessible to the general public is currently missing from the vector tiles.

It would be convenient if the access restrictions would be conflated somehow, as on OSM, the `access` schema is quite complex. E.g. relevant for car drivers is not only `access` but also `motor_vehicle`.

**Comment by pnorman** [comment]
It's not too hard to process the access schema server-side to derive access for car/pedestrian/bike. Generally for rendering you want to simplify access levels into yes, some cases, and not the general public. It's not obvious if we should do that or the styles should.

We also want to avoid having three access attributes on every single road - we don't need to know that cars are allowed on a residential road because that's the norm and to include that property on every road would inflate tile size.

**Comment by westnordost** [comment]
Right, I think a single `access` value that applies for the expected vehicle type would be fine for the purpose of displaying a restriction on the map. Suggestions:

1. For all roads down to service roads (but except pedestrian roads), only the `access` and `motor_vehicle` key is considered. This means that if there is an access restriction for trucks only, this information is not included in the layer, which is fine for general purpose base maps, IMO

2. For values...

- `designated` coalesces to **_yes_**

- `permissive` coalesces to either **_yes_** or **_limited_**. I think the latter would be better

- `customers`, `destination`, `agricultural`, `forestry`, `delivery`, `discouraged` coalesces to **_limited_**

- `permit` coalesces to **_limited_**. I think it is mostly used for residential parking (residential parking permits). So, in that regard a bit similar as `destination` for roads.

- (bicycle, horse, ... etc) `dismount` coalesces to **_no_**

- `military`, `private` coalesce to **_no_**

- all other values are ignored

**Comment by pnorman** [comment]
I'm +1 on bringing this in to 1.1

**Comment by joto** [comment]
I like the proposal and the simplification, but I am not sure whether it is meant as only addressing the "car" use case. I think we should at least use names that would allow later extensions to other modes of transport or have the "bike" and "pedestrian" use case straight away. "horse" is probably too niche, "hgv" also too specialized, but it should be possible to add these in an extension without any clashed in naming or so.

Can we have a properly made out wording before voting on this?

**Comment by pnorman** [comment]
For all kind values coming from a highway tag have
- car_access=yes/limited/no/omitted
- bicycle_access=yes/limited/no/omitted
- foot_access=yes/limited/no/omitted

The keys for the access tags are first turned into one of the values below

| Access value | Attribute |
|-|-|
|yes | yes |
| designated | yes | 
| permissive | limited |
| customers | limited |
| destination | limited |
| agricultural | limited |
| forestry | limited |
| delivery | limited |
| discouraged | limited |
| permit | limited |
| dismount | no |
| military | no |
| private | no |
| no | no |

Other values are removed.

The first access attribute from the table above is taken in the order below.
- car: `motorcar`, `motor_vehicle`, `vehicle`, `access`
- bicycle: `bicycle`, `vehicle`, `access`
- foot: `foot`, `access`

The attribute is taken from the table below
